Changing wording and syntax in Hai Revealed 1 - 4 (#7512)
endless-sky/endless-sky: data/hai/hai reveal 4 middle.txt

				goto questions
			label cadmium
			`	A Hai in a medical uniform explains, "The victims each seem to have been exposed to airborne particulates containing cadmium which must be circulating through their ships ventilation. I should also clarify that it's not strictly 'poisoning'. The level of exposure is quite low, but enough that testing shortly after being rescued does show clearly elevated levels in many who were taken.`
			`	"No known technology of human or Hai origin exhausts anything which could create this effect. There are some microbots used in Hai medicine that are used to seal off ruptures causing internal bleeding. Their dispenser canisters have a residue of residual cadmium dust in them, but those are the only thing we even bring onto a ship that uses it and are self-contained for disposal. The molecular bonding doesn't match anyway. There are some materials involved with properties unlike anything either of us make, which suggests this is coming from another race's technology."`
			`	They share a glance with Teeneep, "We have our suspicions but they're not conclusive yet."`
			branch questions
				not "00 Hai Leaks Response 4B: declined"

> 			`	"We did, however, get a hold of the scan results from the Geocoris you rescued. Interestingly, the self-repair systems it has appear to use a technology which could be responsible for something like this, and we suspect they came from the same place."`

Author: I'm not sure semicolons are used this way.

> 			`	"We did, however, get a hold of the scan results from the Geocoris you rescued. Interestingly, the self-repair systems it has appear to use a technology which could be responsible for something like this, and we suspect they came from the same place."`

Reviewer: Iirc, both are correct, but it's a matter of regional English. The way Grey uses it being considered (out)dated in US English, and elsewhere it may still be considered the sole proper form.

		event "alondo symbol timer" 12
		log `Been asked to check out the Hai spaceports in various systems near the wormhole, just to check nothing unexpected is happening.`
		conversation

> 			`You make what is beginning to feel like a standard request to land at the secure spaceport and disembark with relevant copies of files on a data stick in hand. It doesn't take long to track down Alondo, though you do end up having to wait for nearly half an hour while he finishes up a meeting with a Hai official of some kind. When he's finished he waves you into the room and gestures for you to take a seat. The seat you sit in is clearly designed to be ergonomically comfortable for a Hai to sit in, which makes for a rather strange experience for a human.`

Author: I had a hard time with this bit. Yours looks much better.

			
			label negotiate
			`	The voice laughs. "Ah, yes. Those squirrels. I take it they've had enough of us plundering their systems. Boys!"`

> 			`	The floodlights turn off, revealing a large room with what must be hundreds of pirates with guns of various sizes trained on your ship. One wrong move and you're toast. "Please, captain. Step out of your ship."`

Author: I think he wants to sound condescending, so Scar would sarcastically refer to the player formally than not. I will revert this one.

			
			label refused
			`	"This is most unfortunate," another elder says. "We would have been willing to provide them the technology they sought if only they had asked nicely in the first place."`

> 			`	"Better that we eliminate a threat instead of making them stronger through becoming their weapons dealer," the first elder says. "These pirates might have attacked us with our own technology - or worse still, attacked other humans with them, making us complicit in their slaughter."`

Author: Fair enough. I use dashes a lot, but it doesn't seem like the rest of Endless Sky has many, so I'll go along with you.

	source Darkwaste
	destination Hai-home
	passengers 1
	blocked `It's probably not advisable to bring so many unnecessary people to a secret location on Darkwaste. You should return when your spare bunks are unoccupied.`
	to offer
		has "event: alondo symbol timer"
		or
			has "Hai Reveal [A04] Report to Teeneep: done"
			has "Hai Reveal [A03b] Report to Teeneep: offered"
	on visit
		dialog `Teeneep is on another ship not yet in the system. Take off and wait for the ship to arrive in order to continue.`
	on offer
		conversation
			`Teeneep is waiting for you when you return to her base on Darkwaste, though notably the security point at the front is properly manned this time. Perhaps the most recent raid has given them cause for concern. There is a quiet corner of the living area that is unoccupied at the time, and she gestures for you to join her there.`
			`	"So," she says, "please, tell me everything about this recent experience. I must know what happened and what the Elders chose to do."`

> 			`	You briefly relay the initial events, starting with the raid, the request for your assistance, then the search that you went on, and the eventual discovery of Scar's Legion and their secret base in Danoa where they issued their demands.`

Author: Yes, more succinct.

				`	"I thought even pirates wouldn't blow up relays?"`
				`	"Why would they be doing that, don't they use them too?"`
			label relays

> 			`	Remington nods. "Yes, well, this deep end of space has never had relays. No one out here has needed them; and for a long time the marauders here have been using the communications darkness out this way to be able to get around and do their business with impunity. There are no worlds disconnected, so there's no benefit to them. The only disadvantage is that ships out this way can then call for help when they couldn't before. So the relays the Syndicate have been putting up have been getting destroyed, along with several of our security ships.`

Author: I saw this one last night but it was too late for me to change it, and it definitely looks like an advantage to me.

			`	Alondo looks at you and says, "We need you to do this. It'll be very helpful."`
			`	The emphasis he places on the words 'you' and 'helpful' suddenly give you the sense that this might be coordinated specifically for your benefit. Realizing this, you nod and accept it, on his account.`
			label accepted
			`	"Excellent," says Remington, standing up and flicking over the pickup details to your pad. "Check in with Terry on your way; she may have updated scouting information on their precise placement. Now I have another meeting to get to momentarily, but thank you, Captain. It's been a pleasure seeing you again."`
			`	He gives a meaningful look to Alondo on his way out, but exactly what its meaning is, is not clear to you. Alondo's posture relaxes somewhat as he briefly gets up and quietly closes the door.`

> 			`	"Now that that's over with, we have business. I have a lead on this symbol you wanted investigating. I still wish you'd tell me why you're looking for it, but for now, I'll continue to accept that it's important." he fishes around in a small satchel bag that usually carries his pad and unexpectedly pulls out a piece of physical paper. The text on it reads, 'Adjunct Research Library, Navy Intelligence Observatory, Moscow. Room number: 3017A19'.`

Author: I suppose the paperless office came 1000 years later than people thought it would. It does sound better, and because the Endless Sky world isn't really all that different from ours, we need more of these little things that make it different.

			`	"This David Cohen character is an eccentric one, but things are progressing rapidly and smoothly so far. The station is working, and the cover is gaining traction. From what I hear they're even actually making the movie now. They've almost made enough progress that people could believe the movie has been a work in progress for long enough to justify all this."`
			choice
				`	"You certainly seem busy."`
					goto busy
				`	"Looks like it's thanks to you."`
			`	She momentarily adopts a self-satisfied smile.`
			label busy
			`	"Yes, well, this poses a few issues of its own with the increased demand on the relay bandwidth from here, but I wouldn't have my regular job if something like this was too much to stay on top of."`
			choice 
				`	"Your regular job?"`
					goto regular
				`	"Your job as a secretary?"`

> 			`	She rolls her eyes and shakes her head with a little huff. "Ridiculous, that. It's what Sam tried to have me doing. No, I mean the job I had at the Syndicate right before the end of the war; the job I still seem to be doing seventy percent of even while I'm here."`

Author: Yeah, rereading it I can see what you mean. But, I couldn't tell the first time, and I think others wouldn't have been able to either.

> 			`	She rolls her eyes and shakes her head with a little huff. "Ridiculous, that. It's what Sam tried to have me doing. No, I mean the job I had at the Syndicate right before the end of the war; the job I still seem to be doing seventy percent of even while I'm here."`

Reviewer: Yea, I didn't read it spoken like this either at first, I figured it was a double print on the word.

People DO speak this way, but it's not at all good form in writing. I'm really not sure of a way to make it more clearly read as it would be spoken.

I'd suggest leaving this dangling for now, see if one of the other reviewers like @W1zrad has a better grasp on the technical form?

> 			`	She rolls her eyes and shakes her head with a little huff. "Ridiculous, that. It's what Sam tried to have me doing. No, I mean the job I had at the Syndicate right before the end of the war; the job I still seem to be doing seventy percent of even while I'm here."`

Author: I've changed it to something similar to the original but I don't have much to make it look better. It's still a bit clunky.
>She rolls her eyes and shakes her head with a little huff. "It's ridiculous that Sam tried to make me look like some common witless secretary. No, I mean the job I had right before the end of the war, and the job I still seem to be doing seventy percent of even while I'm here."

	to offer
		has "Hai Reveal [A07] Deliver the Relays: done"
	on visit
		dialog `Batu is on another ship not yet in the system. Take off and wait for the ship to arrive in order to continue.`
	on offer
		log "People" "Giti Gilani" `Giti is a research librarian on Earth that specializes in the history of space flight and related archaic topics.`

> 		log "Factions" "Base Principles" `A clandestine organization that existed within the Syndicate that was founded upon anarcho-syndicalist ideals. Its ranks eventually became bloated by enthusiastic recruits from Freedom that were keen to get rich on top of their freedom, and were more anarchist than anarcho-syndicalist. The resulting tensions undermined the group, and Syndicate Internal Affairs scoured them from the organization.`

Author: I'm not sure about this one because it is in the past. I would also remove "on top of their freedom" because it is a repetition to the planet Freedom.

		conversation
			`You return, giving Batu directions to Xilin, and check availabilities. The other diplomats all appear to be busy and so you soon find yourself at Sayari's office only to be met by Teeneep and Sayari arguing. You came to get an update and instead walked into a very tense standoff, with Sayari standing in front of the door and brandishing a drink bottle in one hand while Teeneep shouts in Hai and gestures at various datapads. It looks like Sayari is doing her best to prevent Teeneep from leaving, though it's hard to say if she'd actually be capable of stopping the much younger Hai physically.`
			`	Realizing that you've arrived Sayari tilts her head slightly to one side to indicate she's speaking to you, while not taking her eyes off Teeneep. "Welcome back captain <last>, I must apologize for my friend's state. She's just discovered some rather unfortunate information."`

> 			`	"Unfortunate!" Teeneep very nearly shrieks the word. "They've been doing this for decades! Decades, Sayari! Tracking down and ultimately paying off pirates with our technology to leave us alone, even when they've hurt, killed, kidnapped, and enslaved our people!"`

Author: Yes, but all have been happening at the same time by (what seems to be) multiple groups, and kidnapping can end in death too. If you want to keep it this way I will add an "and" between "hurt, killed".

			`	Sayari looks at you apologetically. "I promise, we normally get along very well." She turns back to Teeneep. "Look, I'm not saying you shouldn't do something about it. I'm just saying your first course of action shouldn't be to parade it across every network in the whole damn cluster."`
			choice
				`	"What should she do?"`
					goto do
				`	"Does Hai media not do that?"`
			`	Sayari gives you a quizzical look, "Of course they do, though Tee has access to more efficient options. That's not the point, though."`
			label do

> 			`	Teeneep seems to be taking a moment to breathe and Sayari takes that opportunity to lower the drink bottle and let you into the room, closing the door again behind you. "The grievance is against the Elders. There's a process to seek redress of grievances with the Elders. Teeneep needs to at least attempt to follow that process, and if she does not go through with normal resolution channels, she will be censured from public life. She must confront the Elders directly."`

Author: I think your new one looks good enough.

			`	Back in Sayari's office Teeneep seems to have shifted from anger to impotent frustration. "We don't want to risk larger problems with humanity," she imitates a deep stuffy voice. "You're already risking larger issues with your current actions," she whines, speaking with one hand while rolling her eyes at the imagined Elder on her arm.`
			`	"Do they even hear themselves? The Hai expect their Elders to be wise." She scoffs, "You'd find more wisdom in a kindergarten."`
			`	Sayari gives you a long-suffering look but humors her friend. "You're right, of course, in this matter. The Elders have ever struggled with change. They're not wise for this, they were not elected to be so. They were elected to continue the long march of incremental development. Besides, you know far better than most exactly what challenges they contend with these past couple decades. More of them have died or retired than been unelected."`
			`	Teeneep looks grumpy but doesn't contradict the older diplomat.`
			`	"Besides, we know our next steps. Deputy Minster Yang will be here in a moment and we can see what the Republic might do."`

> 			`	Only moments later, Xilin arrives with Batu in tow. While occasionally consulting Teeneep for specific information and details, Sayari, as the actual diplomat, lays out the concerns to do with the Navy knowing of and willfully not intervening. Xilin sympathizes, and, when it comes to deciding what to do, the Hai request that the Navy be deployed to rescue those captured. Before Xilin can commit anything, however, Batu cuts in. "I'm sorry, deputy minister, but I must interject. It is unfortunate I have not had time to fully brief you on everything I've been told before coming, but I have been advised that the Navy will not be deployed to clean up any of the Hai's messes."`

Author: Xilin is still his first name here, it's how Chinese names work.

			choice
				`	"Well, what now?"`
				`	"That's a long time."`
			`	Teeneep sighs in exasperation. "I won't wait for your parliament to spend months deliberating. Sayari, you have a crisis of your own here, and I cannot accept a resolution without an avenue to rescue our people. I would hope you cannot either... I must go to the Unfettered. If they can be swayed then the Elders may be pressured into acting.`
			`	"Captain <last>," she says, giving you a calculating look. "I would very much appreciate it if you'd be so kind as to take me."`
			`	"Teeneep, no! You cannot just commandeer the Captain to act as your chauffeur wherever you want to go," says Sayari. "They have an important role here, in this crisis."`
			# Teeneep yes.
			`	"And what would you have them do next then?" When Sayari seems at a loss to respond, Teeneep continues. "Exactly. Your crisis is mostly resolved behind closed doors, but this crisis requires transportation. The thing you seem to be missing is that they're the same crisis, Sayari. They can't be resolved separately."`

> 			`	Sayari slumps back in her chair with a huff, for a moment looking as old as she says she is. Teeneep waits patiently while her friend spends a good thirty seconds seriously thinking about something. "She is right, <last>," says Sayari eventually. "This is worth trying. Please take her to the Unfettered if that is what she wishes; their government is on <destination>, but please, be careful."`

Author: It might be how they're saying it, without pausing, but it doesn't look right when it's written down and I think it ends up looking ameteurish. I would rather leave the commas in.

> 			`	Sayari slumps back in her chair with a huff, for a moment looking as old as she says she is. Teeneep waits patiently while her friend spends a good thirty seconds seriously thinking about something. "She is right, <last>," says Sayari eventually. "This is worth trying. Please take her to the Unfettered if that is what she wishes; their government is on <destination>, but please, be careful."`

Reviewer: I agree here, and that's why I added the commas in. ;)

But, in the end, spoken word doesn't have to follow grammatical rules, and there's precedence for breaking this specific convention in other ES conversations as well.

It is just cropping up -a lot- here in HR

			`	The Unfettered leader responds, "Your people have outlawed and abandoned us. You give us insulting food donations with propaganda on them. You block our ships from returning to Hai-home. Why should we help you, a race of slothful cowards? Your people are willingly blind and lost, an insult to all Hai."`
			`	Teeneep suddenly takes three strides forward, closing the distance between her and the Unfettered leader in less than a second. You had forgotten how quick Hai are; few humans could do what she just did. "I challenge you for rule of the Unfettered military!"`
			`	"You haven't the authority to do that," he responds, laughing.`
			`	"Besides..." you see one of the other Unfettered to the right of the leader begin to step out. "You would ha..." Teeneep cuts the leader off as she draws a throwing knife and launches it. It zips through the air, and spears the shoulder of the Unfettered who began to move, embedding itself in the wall behind and pinning them in place.`

> 			`	"I would have to do what, defeat your right hand first?" She takes a step back and smirks at him. "Now, will you face a woman half your size?" From her expression, you're unsure if she genuinely has that much faith in her abilities, or if this is a bluff.`

Author: I think "do what" sounds better here.

#		log ``
		conversation
			`You are granted access to the secure spaceport, potentially because nobody has filed any paperwork to stop you, and you can't help but notice that the strangely-outfitted Centipede from Teeneep's operations on Darkwaste has made its way here as well. You have arrived late in the day and there is a gloom developing as the sun disappears behind the buildings of the capital as you enter into what you have begun to think of as the 'diplomatic wing' of the government complex. Fortunately, though the place appears nearly deserted, you swiftly run into Alondo and Remington who are sitting in the empty cafeteria area with paired glasses of what appears to be scotch whiskey.`

> 			`	When he sees you Alondo raises a hand and gestures for you to sit with them. "Captain <last>! Bloody hell, come join us."`

Author: Fair enough. I wonder why the player is never called by their first name in this entire story. It's only stuck out to me just now.

> 			`	When he sees you Alondo raises a hand and gestures for you to sit with them. "Captain <last>! Bloody hell, come join us."`

Reviewer: The drunken breach of formal rank would be a great place to swap in <first> here, imo, yea.

			label continue
			`	Alondo gestures at the large, wall-mounted monitor they've been watching, and you realize that it must be displaying some kind of post-election coverage. The writing is all in Hai, but it seems that the majority of the information that would matter to the diplomats has been displayed visually. On its own you'd have never guessed at trying to read Teeneep's name written in Hai, but when it's presented next to her face and then shown on a pie chart next to percentages and other graphics, it becomes easy enough to follow what's happening.`
			`	"Every so often they have a segment in something we can understand, presumably because of the rest of us out there," says Remington, gesturing vaguely beyond the walls to the outside. "You wouldn't believe some of the rhetoric the last few days while you've been gone."`

> 			`	Alondo visibly trembles, and pushes the bottle across the table back to Remington. "Sometimes it was like watching the old documentaries and political dramas I studied back at school. The ones where there's some populist extremist with dangerous ideas that the establishment is just blindly assuming couldn't possibly be successful, while at the same time that same populist is whipping crowds into a violent frenzy on the streets outside."`

Author: If you'd like. Shudder has become one of those words that looks too much like a meme to me, but it's not a big deal.

> 			`	Alondo appears to tremble, and pushes the bottle across the table back to Remington. "Sometimes it was like watching the old documentaries and political dramas I studied back at school. The ones where there's some populist extremist with dangerous ideas that the establishment is just blindly assuming couldn't possibly be successful, while at the same time that same populist is whipping crowds into a violent frenzy on the streets outside."`

Author: I just don't like the word shudder. I'll change it to "Alondo visibly trembles".

			`	He manages to make that last line sound both bewildered and grumpy at the same time. "That Teeneep you've been running around with - and yes, don't look at me in shock, Sayari told me when we were concerned you might not get out. You've developed some friends in high places and I admire that... where was I?"`
			`	"Teeneep!" says Alondo, perhaps a little too loudly, nearly poking Remington in the face.`
			`	"Of course, that Teeneep of yours, she had her position settled yesterday. She's being..." He curls his lip and frowns while looking for the right word. "Inaugurated! She's being inaugurated in about an hour or so. It's why we're still here. She's the youngest 'elder' since before we developed fire, or some similarly ridiculous time-frame. She seems to have the most influence among the new lot, at least now that sir pitchforks and pulse rifles down there has definitively lost. She's a firebrand, that one, but she seems principled and decent. We're waiting to hear what she has to say in her victory speech."`
			choice
				`	(Wait with them.)`
				`	(Drink with them.)`

> 			`	As it turns out, Teeneep's speech is in Hai with no translation available. It's short, punchy, but restrained; or at least her delivery of it had that feeling. With nothing else to do, and it being well into the evening, you call it a night.`

Reviewer: ```suggestion
			`	As it turns out, Teeneep's speech is in Hai with no translation available. It's short and punchy, but restrained; or at least her delivery of it has that feeling. With nothing else to do, and it being well into the evening, you call it a night.`
```

Transition in tense from "is" in "it's" to "had" is no good.

Author: I had thought that the player was looking back on it as it happened before, because the player is calling it a night right now. But I would have had to change the description above too, and I didn't.

			`	"Well, the good news is that we're probably not going to be getting into a fight with the Republic. Peaceful voices for change triumphed over more radical instigators. The new Elder council has somewhat more, uh, flexibility than the previous one."`
			`	Xilin actively seems to draw strength from this news, and Remington claps Alondo on the shoulder with a quiet, "told ya." Sayari continues. "The bad news is that I'm afraid you're all going somewhere. You have an hour to collect your things and be ready to go. We have a fleet to meet and there's more work ahead of us."`
			label summary
			`	The freshest of the group, Batu, seems to twig to something first. "There's going to be an ultimatum, isn't there? The fleet is a threat."`

> 			`	"Yes," says Sayari. "It's going to be a very broad ultimatum though, and lots of room for the Republic to decide how it's addressed; and no one is shooting at anyone without talking first. Now, all of you except <last> are traveling with Elder Teeneep. I suggest you go and prepare immediately."`

Reviewer: ```suggestion
			`	"Yes," says Sayari. "It's going to be a very broad ultimatum though, with lots of room for the Republic to decide how it's addressed; and no one is shooting at anyone without talking first. Now, all of you except <last> are traveling with Elder Teeneep. I suggest you go and prepare immediately."`
```

> 			`	She approaches you as they leave. "Things might be tense now, but I promise, things aren't as bad as they look. I know Teeneep, and she has the influential backing of another long-time advocate for sensible change elsewhere who was also elected, and who is remaining here to reinforce her position. Teeneep displaced that Elder who threatened her in the chambers. It's upset all the long-term power dynamics in the Council, but it's a much better Council overall now. You're to come with us too, and I'm told I'm required to travel with you, as insurance."`

Reviewer: ```suggestion
			`	She approaches you as they leave. "Things might be tense now, but I promise, things aren't as bad as they look. I know Teeneep, and she has the influential backing of another long-time advocate for sensible change elsewhere who was also just elected. They are remaining here to reinforce her position. Teeneep displaced that Elder who threatened her in the chambers. It's upset all the long-term power dynamics in the Council, but it's a much better Council overall now. You're to come with us too, and I'm told I'm required to travel with you, as insurance."`
```

> 			`She hesitates seeming oddly nervous for the one who theoretically has all the power in the room. "<last>, this will sound silly, but I'm a bit of a traditionalist sometimes and there's a superstition about space travel, and under the circumstances... I'm too old for this stress. Could you please do me a favor and make sure you have a Quantum Keystone on your ship? It's meant to be good luck and well. I think we need it."`

Reviewer: ```suggestion
			`She hesitates, seeming oddly nervous for the one who theoretically has all the power in the room. "<last>, this will sound silly, but I'm a bit of a traditionalist sometimes. There's a superstition about space travel, and under the circumstances... I'm too old for this stress. Could you please do me a favor and make sure you have a Quantum Keystone on your ship? It's meant to be good luck, and, well, I think we need it."`
```

			`	Xilin nods grimly. "However, that would be a drastic failure of foreign policy. It would undermine the legitimacy of the Republic, especially after the war we've just had, and it would be impossible to realistically assure people that an invasion is really not going to happen. True or not. In all likelihood The Republic would fracture and the Free Worlds would probably be on the receiving end of a migration crisis. Parliament has tried to say that they'll take care of it themselves, but the Hai say they've had enough of sitting around and want to be there to save their own people.`
			`	"They haven't outright said it... but they've implied that the Navy could end up killing the Hai involved in the process by not understanding the capabilities of the Hai, or by placing too great an emphasis on preserving Navy lives. Enough people in the Cabinet disagree, that we're sort of at an impasse, for the moment."`
			`	"Who in the Cabinet is blocking it?" The speaker is Turner, and everyone turns to look at him with a mixture of irritation or surprise.`

> 			`	Xilin proceeds to list of a number of names and after a few Turner cuts in again. "Remington, I know you're the PR guy and you keep your nose out of sales, so I apologize if it's presumptuous for me to assume you haven't considered this, but, most of those names are people who listen to the Syndicate. I should know, I've had to negotiate with them as a bloc often enough for government contracts. Has anyone on your side reported back to the Syndicate Board and suggested that perhaps, if we can influence them to support a solution that we propose, even if we don't have that solution defined yet, the Hai could be a very large and very profitable market for Syndicate goods?"`

Reviewer: ```suggestion
			`	Xilin proceeds to list off a number of names, and after a few Turner cuts in again. "Remington, I know you're the PR guy, and you keep your nose out of sales, so I apologize if it's presumptuous for me to assume you haven't considered this, but, most of those names are people who listen to the Syndicate. I should know, I've had to negotiate with them as a bloc often enough for government contracts. Has anyone on your side reported back to the Syndicate Board and suggested that perhaps, if we can influence them to support a solution that we propose, even if we don't have that solution defined yet, the Hai could be a very large and very profitable market for Syndicate goods?"`
```

> 			`	There is a long pause as Remington looks to Terry and they engage in some kind of silent exchange before he replies. "You know, I don't think we'd connected those two specific dots just yet." He looks at Terry. "We need to make some calls."`

Reviewer: ```suggestion
			`	There is a long, expectant pause as Remington looks to Terry and they engage in some kind of silent exchange before he replies. "You know, I don't think we'd connected those two specific dots just yet." He looks at Terry. "We need to make some calls."`
```

This'll preserve the meaning while still avoiding the word that most won't recognize as appropriate to the context.

			# If absolutely necessary you could break this conversation here and have a short mission. Anything more than a couple days strains believability though.

> 			`	The next several hours pass slowly as they do their work, support in the Cabinet is gained, and the data sets collected are assessed for potentially viable priorities. Towards the end of it, you're not really in the loop anymore and the mechanisms of government simply don't include you, but after a while Batu comes to find you. "Come on, there's a meeting between the President and the Elders. You should come and join the witnesses."`

Reviewer: ```suggestion
			`	The next several hours pass slowly as they do their work, support in the Cabinet is gained, and the data sets collected are assessed for potentially viable priorities. Towards the end of it, you're not really in the loop anymore. The mechanisms of government simply don't include you, but after a while Batu comes to find you. "Come on, there's a meeting between the President and the Elders. You should come and join the witnesses."`
```

			`	The smoothness with which this goes down gives you the sense that the main points of this were all provisionally agreed to already, with only a little wiggle room left to be settled in this conversation.`
			`	Sayari steps up and whispers something to Teeneep who then seems to think for a moment before returning to the screens. "This pathway is acceptable."`
			`	A number of people involved in the busywork of bureaucracy start slipping out of the room, doubtlessly to get a head start on certain preparations as Teeneep and the leaders on the screen conclude some formal thanks for continued cooperation and other matters of protocol. You are too visible to slip away though and have to wait.`

> 			`	When you finally emerge Terry and Turner are waiting for you. Terry addresses you. "Captain <last>. We have a little problem. The many civilians that have been living under this cover or being serviced by it have been kept in the dark, and are very nervous about the massed Navy ships. They've been so far sold a story about a 'pirate menace' and now we need them out of here."`

Reviewer: ```suggestion
			`	When you finally emerge Terry and Turner are waiting for you. Terry addresses you. "Captain <last>. We have a little problem. The many civilians that have been living under this cover or being serviced by it have been kept in the dark, and are very nervous about the massed Navy ships. They've been so far sold a story about a 'pirate menace,' and now we need them out of here."`
```

Author: Thanks, you caught a good deal! I don't know how you'd like hai revealed 1 then, because I changed more there.
